gems/pending/VMwareWebService/MiqPbmInventory.rb

```diff
@@ -50,20 +50,21 @@ def pbmQueryAssociatedEntity(profile_ids)
     assoc_entities
   end
 
-  def pbmQueryMatchingHub(profile_id)
-    hubs = []
+  def pbmQueryMatchingHub(profile_ids)
+    hubs = {}
     return hubs if @pbm_svc.nil?
 
     begin
-      # If a string was passed in create a PbmProfileId object
-      profile_id = RbVmomi::PBM::PbmProfileId(:uniqueId => profile_id) if profile_id.kind_of?(String)
+      profile_ids.each do |profile_id|
+        # If a string was passed in create a PbmProfileId object
+        profile_id = RbVmomi::PBM::PbmProfileId(:uniqueId => profile_id) if profile_id.kind_of?(String)
```

Comment:
We use RbVmomi?  This also concerns me in that it's not using the connection broker.